### PR TITLE
[#27] [Chore] Setup CD pipeline for web

### LINKED ISSUE #27: Setup CD pipeline for web
## Why

We need to set up a `CD` pipeline to achieve this outcome

>  Deploy the web version to a simple hosting site where it can be accessible by non-engineers.

## Who Benefits?

Developers

### PR DESCRIPTION
**Pull request by markgravity**
- Close #27 

## What happened 👀

- Create a workflow to deploy the web version
- Setup Heroku app
- Add required Github Secrets to run this workflow

## Insight 📝

As we base on this [build_pack](https://github.com/EE/heroku-buildpack-flutter-light) (out of update) and update some changes to make it work, I also submit a [PR](https://github.com/EE/heroku-buildpack-flutter-light/pulls) to add our changes

## Proof Of Work 📹

The web version is up
https://showcase-flutter-build-web-ae273aa1dc1e.herokuapp.com

### CODE REVIEW

**Review by ducbm051291 — APPROVED** [review]
All set 👍

File: .github/workflows/deploy_production.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: CI
```
Comment (Wadeewee) [inline .github/workflows/deploy_production.yml:1]:
```suggestion
name: CD
```

**Review by Wadeewee — COMMENTED** [review]
Rest LGTM 🎉  Well done @markgravity 🙇‍♀️

File: .github/workflows/deploy_production.yml (hunk 1 above)
Comment (markgravity) [reply .github/workflows/deploy_production.yml]: @Wadeewee updated at [99de761](https://github.com/nimblehq/showcase-flutter-build-web-desktop/pull/35/commits/99de761333bbcd0d949d527286fa58f9637f4077)

**Pull request merged**